.codex-supervisor/issue-journal.md

```
@@ -1,37 +1,50 @@
+- Updated at: 2026-03-20T16:43:06Z
 
 ## Latest Codex Summary
-- Added a focused docs regression for workspace-restore precedence, reproduced the missing contract in the English docs, then updated `README.md`, `docs/architecture.md`, `docs/getting-started.md`, and `docs/configuration.md` so they consistently say `ensureWorkspace()` should prefer an existing local issue branch first, then an existing remote issue branch, and only then bootstrap from `origin/<defaultBranch>` as the fallback. Focused docs verification passed, and `npm run build` passed after installing local dependencies in this worktree.
+Addressed the three remaining CodeRabbit follow-ups in [.codex-supervisor/issue-journal.md](./issue-journal.md) without changing any restore behavior. The journal now uses repo-relative links in the latest summary, keeps the review-status narrative consistent with the current branch state, and no longer contains inline-code spacing that could trigger the reported MD038 warning.
+
+The journal-only fix is pushed to `origin/codex/issue-721` at `1707486`, and review threads `PRRT_kwDORgvdZ851wrHP`, `PRRT_kwDORgvdZ851wrHV`, and `PRRT_kwDORgvdZ851wrHX` were resolved after the push. The untracked `.codex-supervisor/replay/` directory was left untouched.
+
+Summary: Fixed the journal-only review fallout, pushed `1707486`, and resolved the remaining CodeRabbit threads on PR #746.
+State hint: waiting_ci
+Blocked reason: none
+Tests: `rg -n '\\]\\(/home/tommy' .codex-supervisor/issue-journal.md`; `perl -ne 'while(/(?<!`)`([^`]+)`(?!`)/g){ print qq($.::<$1>\\n) if $1 =~ /^\\s|\\s$/ }' .codex-supervisor/issue-journal.md`; `git diff --check -- .codex-supervisor/issue-journal.md`
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)